aspnet/fundamentals/dependency-injection.rst

```diff
@@ -1,10 +1,277 @@
-.. _dependency-injection:
+Dependency Injection
```

Comment:
There should be another DI article for MVC. This one should reference it and vice-versa, but there are enough DI-specific parts of MVC that it deserves its own coverage (including the attribute-based stuff you noted below).